### DISCUSSION TITLE
Universal Calculator Assignment

### DISCUSSION THREAD

**Discussion by Souky-11** [Souky-11/TafeT2S2T1VIRGO3#1] [Announcements]
Hi Althea

I have created a remote repository for us to collaborate for Assignment 2. Any ideas on how we should assign the group work? Do you have any preferences of what you might like to do?

**Comment by altheatafe** [comment]
Hi Souky, 

Thank you for setting it up. 

What do you think if we assign the group work like below:
1. Section B (setting up the local & remote repository, development branch) + Section C (Mortgage Calculator)
2. Section C (Currency Conversion) + Section D (merging into development and release branch)
--> Then work on Section G together?

**Reply by Souky-11** [reply]
That's great ! 
I'm happy to work on  Section B and Section C and then we can work on G together.

**Reply by altheatafe** [reply]
Awesome! I'll work on Section C(Currency Conversion) and Section D then
